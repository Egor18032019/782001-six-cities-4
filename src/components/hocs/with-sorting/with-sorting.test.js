import React from "react";
import renderer from "react-test-renderer";

import withSorting from "./with-sorting.js";

const MockComponent = () => {

  return (
    <div>
    </div>
  );
};


const MockComponentWrapped = withSorting(MockComponent);

it(`MockComponentWrapped is rendered correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped
      typeSorting={`Popular`}
      onSortingTypeClick={()=>{}}
    />
  ), {
    createNodeMock() {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
