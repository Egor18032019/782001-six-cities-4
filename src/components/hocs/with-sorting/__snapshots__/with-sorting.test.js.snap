// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MockComponentWrapped is rendered correctly 1`] = `<div />`;
